js/members.js

```diff
@@ -83,3 +135,12 @@ function clickThirdYears() {
 function clickSecondYears() {
     $('#allMembersContainer').html(members['secondYears']);
 }
+
+function clickAllAlumni() {
+    $('#allMembersContainer').html(members['firstAlumnis']);
+    //hide present members batchButton
+    $("#batchButtons button").css("display", "none");
+    //show alumniButton
+    $("#alumniButtons button").css("display", "block");
```

Comment:
Same as above (set ```display: inline-block``` to the container div instead)

Reply:
done